gradle.properties

```diff
@@ -26,5 +26,8 @@ android.enableJetifier=true
 # URL to Klaviyo server
 klaviyoServerUrl=https://a.klaviyo.com
 
+# Klaviyo API Revision
+klaviyoApiRevision=2023-07-15
```

Comment:
This is referring to a backend version, right?

Reply:
Correct